test/startservers.py

```diff
@@ -81,10 +81,6 @@
         9667, None,
         ('./bin/boulder', 'crl-storer', '--config', os.path.join(config_dir, 'crl-storer.json')),
         ('s3-test-srv',)),
-    Service('ocsp-updater',
-        8006, None,
```

Comment:
`//test/prometheus/prometheus.yml` Line 14 can be removed because boulder:8006 refers to this now removed port.